src/components/users/LoginForm.js

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from "react";
+import { userActions } from "../../actions/user.actions"
+import { useDispatch, useSelector } from "react-redux";
+import "../../style/App.scss";
+import "./login-form.scss";
+
+function LoginForm() {
+    // get the alert from the Redux store
+    const alert = useSelector(state => state.alert);
+
+    // the inputs are going to be updated with setInputs function
+    // initial state is empty email and password
+    const [inputs, setInputs] = useState({email: "", password: ""});
```

Comment:
We usually use libraries to manage the forms so you don't have to do it manually, as I've seen, in web they use `formik`